Dockerfile

```diff
@@ -75,6 +75,6 @@ ENV SECRETS /secrets
 RUN mkdir -p $WORKDIR
 RUN mkdir -p $SECRETS
 
-VOLUME [ $WORKDIR ]
+VOLUME [ "$WORKDIR" ]
```

Comment:
extra directory '[' and ']' will be created without the quotation mark, see https://docs.docker.com/reference/builder/#volume